tests/redirects/test_urls.py

```diff
@@ -72,3 +72,112 @@ def test_301_urls(url, base_url, follow_redirects=False):
 )
 def test_302_urls(url, base_url, follow_redirects=False):
     assert_valid_url(url, base_url=base_url, follow_redirects=follow_redirects, status_code=requests.codes.found)
+
+
+@pytest.mark.headless
+@pytest.mark.nondestructive
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    "url,accept_language_header",
+    [
+        # Mozorg privacy
+        ("/en-US/privacy/", None),
+        ("/en-US/privacy/", "*"),
+        ("/en-US/privacy/", "de"),
+        ("/en-US/privacy/", "br-PT"),
+        ("/privacy/", None),
+        ("/privacy/", "*"),
+        ("/privacy/", "de"),
+        ("/privacy/", "br-PT"),
+        # Fx privacy
+        ("/en-US/privacy/firefox/", None),
+        ("/en-US/privacy/firefox/", "*"),
+        ("/en-US/privacy/firefox/", "de"),
+        ("/en-US/privacy/firefox/", "br-PT"),
+        ("/privacy/firefox/", None),
+        ("/privacy/firefox/", "*"),
+        ("/privacy/firefox/", "de"),
+        ("/privacy/firefox/", "br-PT"),
+        # Focus privacy
+        ("/en-US/privacy/firefox-focus/", None),
+        ("/en-US/privacy/firefox-focus/", "*"),
+        ("/en-US/privacy/firefox-focus/", "de"),
+        ("/en-US/privacy/firefox-focus/", "br-PT"),
+        ("/privacy/firefox-focus/", None),
+        ("/privacy/firefox-focus/", "*"),
+        ("/privacy/firefox-focus/", "de"),
+        ("/privacy/firefox-focus/", "br-PT"),
+        # Mozilla VPN privacy
+        ("/en-US/privacy/mozilla-vpn/", None),
+        ("/en-US/privacy/mozilla-vpn/", "*"),
+        ("/en-US/privacy/mozilla-vpn/", "de"),
+        ("/en-US/privacy/mozilla-vpn/", "br-PT"),
+        ("/privacy/mozilla-vpn/", None),
+        ("/privacy/mozilla-vpn/", "*"),
+        ("/privacy/mozilla-vpn/", "de"),
+        ("/privacy/mozilla-vpn/", "br-PT"),
+    ],
+)
+def test_privacy_policies_always_200_OK(
+    url,
+    base_url,
+    accept_language_header,
+):
+    """Smoke test to ensure that our privacy pages always
+    ultimately return a 200 OK response, even if the client
+    requesting them lacks an Accept-Language header and there
+    is no locale in the actual URL
+    """
+
+    req_headers = {}
+    if accept_language_header:
+        req_headers["Accept-Language"] = accept_language_header
+
+    assert_valid_url(
+        url,
+        base_url=base_url,
+        req_headers=req_headers,
+        follow_redirects=True,
+        final_status_code=requests.codes.ok,
```

Comment:
This looks like the default so not needed. But fine if you want to keep it to be explicit also.